service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,318 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <!--    Assignment 2-->
+
+    <!--    Created Order-->
+    <service verb="create" noun="Order" displayName="Create an order">
+        <in-parameters>
+            <parameter name="orderName" required="true"/>
+            <parameter name="currencyUomId" default-value="USD"/>
+            <parameter name="salesChannelEnumId"/>
+            <parameter name="statusId" default-value="orderPlaced"/>
+            <parameter name="productStoreId"/>
+            <parameter name="placedDate" required="true"/>
+            <parameter name="approvedDate"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId"/>
+        </out-parameters>
+        <actions>
+            <!--Creating Orders in the OrderHeader -->
+            <service-call name="create#mantle.order.OrderHeader" in-map="context" out-map="context"/>
+        </actions>
+    </service>
+
+    <!--    Added Order Items-->
+
+    <service verb="add" noun="OrderPart" displayName="Add an order">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+            <parameter name="partName"/>
+            <parameter name="facilityId" required="true"/>
+            <parameter name="shipmentMethodEnumId" default-value="ShMthGround"/>
+            <parameter name="customerPartyId"/>
+            <parameter name="item_details" type="List">
+                <parameter name="orderMap" type="Map">
+                    <parameter name="productId" required="true"/>
+                    <parameter name="itemDescription" required="true"/>
+                    <parameter name="quantity" required="true"/>
+                    <parameter name="unitAmount" required="true"/>
+                </parameter>
+            </parameter>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId"/>
+            <parameter name="orderPartSeqId"/>
+        </out-parameters>
+        <actions>
+            <!-- Check if the OrderHeader is existing for the given orderId -->
+            <entity-find-one entity-name="OrderHeader" value-field="orderHeader">
+                <field-map field-name="orderId"/>
+            </entity-find-one>
+            <if condition="orderHeader">
+                <then>
+                    <!--If orderId exit then create else return an error -->
+                    <service-call name="create#mantle.order.OrderPart" in-map="context" out-map="context"/>
+                </then>
+                <else>
+                    <return error="true" message="Order doesn't exists." type="danger"/>
+                </else>
+            </if>
+            <set field="orderPartSeqId" from="orderPartSeqId"/>
```

Comment:
From where we are getting orderPartSeqId to set here?

Reply:
It is coming form context